grammars/ooc.cson

```diff
@@ -445,7 +445,7 @@
         'name': 'constant.numeric.float.ooc'
       }
       {
-        'match': '(?x)\\b([0-9]+)(?: [eE][+\\-]?\\d+)?'
+        'match': '(?x)\\b(([0-9]+_?[0-9]+)+)(?: [eE][+\\-]?\\d+)?'
```

Comment:
Is there a way to simplify this? What I'm trying to achieve here is that `_` should be allowed between digits, but not at the start or the end of an `Int` literal, i.e. `123_456_789` is correct, but `_123` or `123_` isn't.